hat/assets/js/apps/Iaso/domains/projects/config.tsx

```diff
@@ -42,7 +42,9 @@ export const columns = (
         accessor: 'feature_flags',
         sortable: false,
         Cell: settings =>
-            settings.value.map(fF => fF.name).join(', ') || textPlaceholder,
+            settings.value
+                .map(fF => formatMessage(MESSAGES[fF.code.toLowerCase()]))
```

Comment:
Please add a check, in case we add a feature flag and don't have a translation. if `MESSAGES[fF.code.toLowerCase()]` is `undefined` we should display `fF.name` and `fF.name` is also undefined then, the id or a placeholder.